templates/careers/index.html

```diff
@@ -18,80 +20,116 @@
   <img src='https://pubads.g.doubleclick.net/activity;xsp=5024608;ord=1?' width=1 height=1 border=0>
 </noscript>
 
-<section class="p-strip--image is-dark is-deep" style="background-image: url(https://assets.ubuntu.com/v1/aebc1742-01_suru2_dark_2K.png); background-position: 100% 55%; background-size: 180%; background-repeat: repeat-x;">
+<section class="p-section">
+  <div class="u-fixed-width">
+    <nav class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
+      <ol class="p-breadcrumbs__items p-text--x-small-capitalised">
+          <li class="p-breadcrumbs__item ">Careers</li>
+      </ol>
+    </nav>
+    <h1 class="p-text--x-small-capitalised u-hide">Careers</h1>
```

Comment:
This heading isn't accessible via screenreader
![image](https://github.com/canonical/canonical.com/assets/17607612/ba72cd55-771f-4244-85be-6d3117a0cd95)

Reply:
mm I don't know why Lyubo added that. Removed.

Reply:
I imagine because the page needs an h1 tag and "careers" was meant to be the most important heading. @marinacastejon are you happy to have the first heading above as the h1 instead?

Reply:
Ok, I've added it back but change to the `off-screen` class instead of `u-hide`